packages/color-kit-block/src/Palette.tsx

```diff
@@ -0,0 +1,39 @@
+import { Stack, Text, Tooltip, TooltipPosition } from '@frontify/fondue';
+
+import { TooltipContent } from './TooltipContent';
+
+import { PaletteProps } from './types';
+
+export const Palette = ({ palette }: PaletteProps) => {
+    const { name, colors = [] } = palette;
+    return (
+        <div data-test-id="palette" className="tw-flex tw-flex-col tw-space-y-2">
+            <Text color="x-weak">{name}</Text>
+            <Stack padding="none" spacing="none">
+                {colors.map(({ id, hex }) => {
+                    if (!hex) {
+                        return <></>;
```

Comment:
`return null;`

Reply:
I think it is better to return a single type instead of multiple. So this component will always return ReactElement, instead of ReactElement | null, as React recognizes Fragment tag as no element.